sound/soc/sdw_utils/soc_sdw_cs42l45.c

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Based on sof_sdw_rt5682.c
+// This file incorporates work covered by the following copyright notice:
+// Copyright (c) 2023 Intel Corporation
```

Comment:
Not sure if the Copyright is needed. And 2025 maybe? @ujfalusi @lgirdwood

Reply:
I think this matches what was done on cs42l42,cs42l42 so I think its probably fine.